Formula/automysqlbackup.rb

```diff
@@ -56,4 +58,7 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
```

Comment:
@GauthamGoli Great, thanks. You can ignore that failure, we'll merge it regardless.